models/au_nsw_bocsar_crime/code/upload.py

```
@@ -0,0 +1,126 @@
+        return expected
+
+
+def main():
+    only = set(_argv)
+    tables = [t for t in TABLES if not only or t in only]
+    print(f"=== uploading to {BILLING_PROJECT} (env={ENV}) ===", flush=True)
+    for slug in tables:
+        print(f"=== {slug} ===", flush=True)
+        try:
+            upload_table(slug)
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)